src/blocks/Listing/components/ListingVideo.tsx

```diff
@@ -35,7 +35,7 @@ const ListingVideo: React.FC<
                                     />
                                     <svg
                                         aria-hidden='true'
-                                        viewBox='0 0 24 24'
```

Comment:
Please add this icon to the icon library :)

Reply:
Done, added to Icon library